hashing/te-tag-query-ruby/TETagQuery.rb

```diff
@@ -585,22 +585,32 @@ def usage(exitCode)
                        HAS_PRIVACY_GROUP these must be comma-delimited
                        privacy-group IDs.
 --tags {...}           Comma-delimited. Overwrites on repost.
+
 --related-ids-for-upload {...} Comma-delimited. IDs of descriptors (which must
                        already exist) to relate the new descriptor to.
 --related-triples-json-for-upload {...} Alternate to --related-ids-for-upload.
                        Here you can uniquely the relate-to descriptors by their
                        owner ID / indicator-type / indicator-text, rather than
                        by their IDs. See README.md for an example.
+
+--reactions-to-add {...}    Example for add/remove: INGESTED,IN_REVIEW
+--reactions-to-remove {...}
+
 --confidence {...}
 -s|--status {...}
 -r|--review-status {...}
 --first-active {...}
 --last-active {...}
 --expired-on {...}
 
-Please see the following for allowed values in all enumerated types:
+Please see the following for allowed values in all enumerated types except reactions:
 https://developers.facebook.com/docs/threat-exchange/reference/submitting
 
+Please see the following for enumerated types in reactions:
+See also https://developers.facebook.com/docs/threat-exchange/reference/reacting
+
+xxx reactions too
```

Comment:
Is this a missed TODO?

Reply:
good catch -- thanks!

Reply:
I'll do this very next commit (promise!) :)

I have another diff in my local repo one ahead of this & my git-fu is weaker than my hg-fu.

Reply:
https://github.com/facebook/ThreatExchange/commit/654836ca119a65324d7726a245c155592b458f8e